shop/rest/renderers.py

```diff
@@ -3,6 +3,46 @@
 
 from rest_framework import renderers
 from rest_framework.compat import template_render
+from rest_framework.exceptions import APIException
+
+from shop.models.cart import CartModel
+from shop.serializers.cart import CartSerializer
+
+
+class ShopTemplateHTMLRenderer(renderers.TemplateHTMLRenderer):
+    """
+    Modified TemplateHTMLRenderer, which shall be used to render templates used by django-SHOP.
+    Instead of polluting the template context with the serialized data, that information is
+    stored inside a separate `data` attribute, which allows to add a Cart and Paginator object.
+
```

Comment:
We have to judge wether it is more important to avoid potential name collisions or to keep the behaviour closer to DRF.